packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.css

```
@@ -3,4 +3,9 @@
 .facet__search-form {
   display: flex;
   align-items: flex-start;
+  justify-content: space-between;
+}
+
+.facet__search-formItem {
```

IIRC we don't use camel casing in our css. 
```suggestion
.facet__search-form-item {
```

---

good to know, will stick with kebabs in the future! 🍢🍢🍢🍢

---

We loosely follow the [BEM naming convention](https://getbem.com/naming/). Or at least we try to... 🙄